packages/patternfly-4/react-core/src/components/Modal/examples/SimpleModal.js

```diff
@@ -14,7 +14,7 @@ class SimpleModal extends React.Component {
 
   render() {
     const { isModalOpen } = this.state;
-    const reactRoot = typeof document !== 'undefined' ? document.querySelector('#___gatsby') : '';
+    const srHide = typeof document !== 'undefined' ? document.querySelector('#___gatsby') : '';
```

Comment:
As an app developer, it's not clear what I should use in place of the '#___gatsby' selector. It would be helpful to add a comment in this example describing what I should use instead.